portfolio/src/main/webapp/style.css

```diff
@@ -1,16 +1,28 @@
+body {
+  background-color:rgb(215, 222, 247) !important;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+
+.nav {
+  background-color: azure !important;
+}
+
+#android {
+  margin-bottom: 4%;
+}
+
 #content, #header, #images{
```

Comment:
May apply here: new lines for each property(? I can't speak css...) https://google.github.io/styleguide/htmlcssguide.html#Selector_and_Declaration_Separation